apps/dashboard/test/fixtures/sys_with_gateway_apps/bc_jupyter/form.yml

```diff
@@ -57,37 +61,52 @@ attributes:
           data-min-bc-num-hours-for-cluster-owens: 80,
           data-max-bc-num-hours-for-cluster-oakley: 99,
           data-min-bc-num-hours-for-cluster-oakley: 90,
+
+          data-min-gpus: 1,
+          data-max-gpus: 4,
         ]
       - [
           "hugemem",
-          data-option-for-cluster-oakley: false,
+          data-option-for-cluster-oakley: "false",
           data-hide-cuda-version: true,
           data-hide-advanced-options: true,
           data-max-bc-num-slots-for-cluster-owens: 42,
           data-min-bc-num-slots-for-cluster-owens: 42,
+
+          data-min-gpus: 0,
+          data-max-gpus: 0,
         ]
       - [
           "advanced",
-          data-option-for-cluster-oakley: false,
+          data-option-for-cluster-oakley: "false",
           data-hide-cuda-version: true,
-          data-max-bc-num-slots-for-cluster-oakley: 9001
+          data-max-bc-num-slots-for-cluster-oakley: 9001,
+
+          data-min-gpus: 0,
+          data-max-gpus: 0,
         ]
       # this node type is the same for both clusters, so there's no 'for-cluster-...' clause
-      # TODO: this currently breaks tests
       - [
           "same",
-          data-hide-cuda-version: true,
-          data-hide-advanced-options: true,
-          data-min-bc-num-slots: 100,
-          data-max-bc-num-slots: 200,
-          data-min-bc-num-hours: 444,
-          data-max-bc-num-hours: 555,
+          # note that all these values are strings
+          data-hide-cuda-version: "true",
+          data-hide-advanced-options: "true",
+          data-min-bc-num-slots: "100",
+          data-max-bc-num-slots: "200",
+          data-min-bc-num-hours: "444",
+          data-max-bc-num-hours: "555",
```

Comment:
These modifications on old tests are just to be sure that when folks use strings instead of integers or booleans everything still works as expected.